docs/API/datasources/query-operators.md

```
@@ -0,0 +1,383 @@
+```
+
+---
+
+## Performance Optimization
+
+### Optimized Operators
+
+The following operators are optimized for better performance with Fliplet's database:
+
+**MongoDB-style (optimized):**
+- `$or`, `$and`, `$gte`, `$lte`, `$gt`, `$lt`, `$eq`, `$in`
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 0

(MD007, ul-indent)